pkg/karmadactl/join.go

```diff
@@ -265,10 +269,6 @@ func generateSecretInMemberCluster(clusterKubeClient kubeclient.Interface, clust
 		return nil, err
 	}
 
-	if dryRun {
```

Comment:
If system is in `dryRun` mode, there will not create resources before.

Reply:
Didn’t see the sub-function and dry-run judgment, thx